package.json

```diff
@@ -72,6 +72,7 @@
     "@storybook/addon-essentials": "^6.5.15",
     "@storybook/addon-interactions": "^6.5.15",
     "@storybook/addon-links": "^6.5.15",
+    "@storybook/addon-viewport": "^6.5.15",
```

Comment:
@lukaw3d any idea why this change would affect CI?

Reply:
yarn.lock seems out of date